shapash/explainer/smart_explainer.py

```diff
@@ -1244,6 +1244,7 @@ def generate_report(self,
                         y_test=None,
                         title_story=None,
                         title_description=None,
+                        palette_name=None,
```

Comment:
palette_name is an attribute of smartplotter.
Can we use it instead of defining a new parameter to this method generate_report ?